etc/env/base/fri_witness_vector_generator.toml

```diff
@@ -1,7 +1,7 @@
 [fri_witness_vector_generator]
 prover_instance_wait_timeout_in_secs=200
 prover_instance_poll_time_in_milli_secs=250
-prometheus_listener_port=3314
```

Comment:
could you allign it with file-based configs. just for making our DevOps team life easier